x-pack/plugins/enterprise_search/public/applications/app_search/components/result_settings/logic/helpers.ts

```diff
@@ -0,0 +1,127 @@
+/*
```

Comment:
Should we rename this to `utils.ts` to match our other files/naming convention?

Also just curious, is there a specific reason why this is in a nested `logic/` folder vs. a top-level file? Are we planning on adding more utils/helpers specific to components later for example?

Reply:
Yes, that is fine. I will rename i. 

I put it under /logic just for additional organization. I'm happy to lift it back up though. I'll probably blow away `/logic` as well, since if it's just the single logic then I think it's unnecessary.

Reply:
e7aae9c725cdb1e90fcc3ea172fc6609e93aeb96